tests/test_slack/test_user.py

```diff
@@ -47,10 +47,10 @@ def test_repr(self):
         assert repr(self.object) == test_user_id
 
     def test_get_user(self):
-        user = SlackUser.get_user(SlackClient('xxx'), test_user_name)
+        user = SlackUser.get_user(DummySlackConnection(), test_user_name)
         assert isinstance(user, SlackUser)
 
 
 def test_get_user_none():
-    user = SlackUser.get_user(SlackClient('xxx'), 'doesnotexist')
+    user = SlackUser.get_user(DummySlackConnection(), 'doesnotexist')
```

Comment:
@arcticfoxnv  can you please review this? I was unsure why it was using SlackClient previously / how it worked.

Reply:
It's been a while since I looked at this.  Not sure why it was using SlackClient.

Reply:
Addressed review comments. Thanks for the feedbac.